src/main/java/appeng/container/implementations/ContainerGrinder.java

```diff
@@ -31,16 +31,16 @@
 public class ContainerGrinder extends AEBaseContainer
 {
 
-	final TileGrinder grinder;
+	private final TileGrinder grinder;
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Remove this unused "grinder" private field. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1068)